### ISSUE TITLE
ckw-b4.zip flagged as virus

### ISSUE THREAD

**Issue by cleeus** [davidrg/ckwin#168]
Hello David,

the [B4 release.zip](https://github.com/davidrg/ckwin/releases/download/10.0beta.08-4/ckw-b4.zip) is consistently flagged as a virus on multiple virus scanners. Virus scanner is complaining about iksdsvc.exe. Could you maybe tag a new build?

Btw, here is the virus total scanner output:
https://www.virustotal.com/gui/file/573dda9df35d4678d7beb736b465349a2c2bec0400d3cf4ce655f0bb299360d7

**Comment by johnsonjh** [comment]
I’ve seen this before too myself - on even the simplest “Hello World” program built with some MSVC versions.  I see it’s been reported:

![IMG_6951](https://user-images.githubusercontent.com/61629094/224976424-889dc0c0-a6ac-4921-94dd-70267c76b40e.jpeg)

**Comment by davidrg** [comment]
Yeah, its a false positive. I put a fresh Visual C++ 14.2 build straight from the github build servers through virustotal with similar (though not as many) results: https://www.virustotal.com/gui/file/856a048aff6fd14f817f082247bdbe4c11428015ac0667a7a390040de3cba60e

This is the main reason the builds for modern windows doesn't include ctl3dins.exe (aside from it not being necessary there) - it kept tripping up my own virus scanner whenever I downloaded a build from github.

Not sure what, if anything, can really be done about it. I assume in this case the virus scanners are just reacting to the capabilities C-Kermit has - making and receiving network connections, transferring files, running as a service, etc.
